documentation/general/general_setup.md

```diff
@@ -1,263 +1,230 @@
 
-A subset of the following requirements will be required depending on the extent
-of your contribution to the project. To see which of these you need, refer to
-the [requirement matrix](#requirement-matrix) above.
+   ```shell
+   ln -s $(git rev-parse --show-toplevel)/ov ~/.local/bin/ov
+   ```
 
-### Python
+   Add `~/.local/bin` to your `PATH` or replace it in the command above with a
+   directory on your `PATH`, and _voilà_, you can run `ov` from anywhere without
+   needing to reference it relative to the repository.
 
-```{note}
-This is only needed if you are working with the following:
+   ```{caution}
+   Throughout the Openverse documentation, we assume `ov` is available for use directly.
+   If you cannot add `ov` to your `PATH`, then use a relative reference to it anytime you
+   invoke the command (i.e., as in `ov init` above).
+   ```
 
-- documentation
-- Python automations
-- API (outside Docker, for debugging purposes)
-- ingestion server (outside Docker, for debugging purposes)
-```
+You're done! Run `ov help` to see a list of useful tools, and visit the
+quickstart guides for each part of the Openverse project in the list below:
 
-We use Python 3 in the backend of our stack. So to work with that part of the
-codebase, you will need Python installed. Most Linux distributions and macOS
-will already have this installed, even if it's not always the latest version. To
-know if you have Python (specifically version 3) installed, run the following
-command.
+```{toctree}
+:maxdepth: 1
+:glob:
 
-```bash
-python3 --version
+/frontend/guides/quickstart.md
+/api/guides/quickstart.md
+/catalog/guides/quickstart.md
+/ingestion_server/guides/quickstart.md
 ```
 
-If you see `Python x.y.z`, you have Python installed. If you see an error, you
-need to install it using [pyenv](https://github.com/pyenv/pyenv) or by following
-the [official instructions](https://www.python.org/downloads/).
+## `ov` usage
 
-#### Pipenv
+Run `ov help` to read basic documentation for how to use and interact with `ov`.
 
-Pipenv helps us provision dependencies for our Python packages and automatically
-set up a virtualenv, provided you have the version of Python mentioned in the
-`Pipfile` installed and accessible locally.
+### Aliases
 
-You can install Pipenv by following the
-[official instructions](https://pipenv.pypa.io/en/latest/installation.html#installing-pipenv).
+`ov` has a number of built-in aliases, which are documented and discoverable by
+running `ov aliases`.
 
-### Node.js
+You may configure your own aliases. Run `ov aliases --help` for documentation.
+`ov init` includes an example alias in the generated `.ov_profile.json` to help
+you get started.
 
-```{note}
-This is only needed if you are working with the following:
+### Included tools
 
-- frontend
-- Node.js automations
-```
+The containerised `ov` environment includes a host of excellent general and
+specific tools to interact with Openverse's development environment. Some of
```

Comment:
Nit: "A number of", or perhaps just "several" might be better to use here so we don't overload the word "host", since we just introduced the phrase "host machine" to some users earlier in this document.